pages/specification/json-hyper-schema/_index.md

```diff
@@ -13,6 +13,531 @@ In essence:
 - JSON Hyper-Schema is helpful in complex APIs where clients need to define and explicitly understand the relationships between resources and actions, especially when navigating resources without prior knowledge of the API structure.
 - It helps create more discoverable and self-documenting APIs, making it easier for clients to interact with them.
 
+
+## Hyper-schema use cases
+
+### Basic concept
+
+**Get a JSON instance**
+
+```http
+GET https://example.com/book/12345
+```
+
+_This `GET` request fetches information about a specific book, identified by the `book` ID (`12345`). In response:_
+
+```http
+200 OK
+Content-Type: application/json
+Link: <https://example.com/schemas/book>; rel="describedby"
+
+{
+  "title": "Mobby-Dick",
+  "authorId": 100,
+  ...
+}
+```
+
+_The server responds with a successful `200 OK` status, JSON-formatted data, and a `Link` header that references the schema describing the structure of the returned data, including details like the `book` `title` and a link to the author via `authorId`._
+
+**Get the schema that describes the instance using the "`describedby`" Link header.**
```

Comment:
issue(style-blocking): remove quotation marks from `describedby`; use code font for `Link` 

Google style guide: https://developers.google.com/style/quotation-marks